compiler/packages/babel-plugin-react-compiler/src/HIR/CollectHoistablePropertyLoads.ts

```diff
@@ -127,6 +133,11 @@ type CollectHoistablePropertyLoadsContext = {
    * but are currently kept separate for readability.
    */
   nestedFnImmutableContext: ReadonlySet<IdentifierId> | null;
+  /**
+   * Functions which are assumed to be eventually called (as opposed to ones which might
+   * not be called, e.g. the 0th argument of Array.map)
+   */
+  likelyInvokedFns: ReadonlySet<LoweredFunction>;
```

Comment:
A subtly different framing for this is functions which must be written as if they will be called unconditionally. Maybe "assumedInvokedFns" or "unconditionallyCallableFns".

Reply:
Updated to `assumedInvokedFns`